lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -709,4 +827,52 @@ sub scalar_property_from_underlying_alignment_results {
 }
 
 
+sub _lock_per_lane_alignment {
```

Comment:
This might be better named `_lock_per_lane_alignments`.  (There's also an argument for `_lock_individual_alignments` since that's what it collects!)